.twig-cs-fixer.dist.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+$ruleset = new TwigCsFixer\Ruleset\Ruleset();
+$ruleset->addStandard(new TwigCsFixer\Standard\TwigCsFixer());
+$ruleset->addRule(new TwigCsFixer\Rules\Function\IncludeFunctionRule());
```

Comment:
Since it's the Symfony code base, I would not recommend using 
```
new TwigCsFixer\Standard\TwigCsFixer()
```
which is opinionated.

And instead use
```
new TwigCsFixer\Standard\Symfony()
```
which is based on best practices from Symfony/Twig docs

Reply:
i see thanks, lets wait some other review and decide which rule set to begin with